lib/sinon/match.js

```diff
@@ -1,428 +1,5 @@
 "use strict";
 
-var arrayProto = require("@sinonjs/commons").prototypes.array;
-var deepEqual = require("./util/core/deep-equal").use(match); // eslint-disable-line no-use-before-define
-var every = require("./util/core/every");
-var functionName = require("@sinonjs/commons").functionName;
-var get = require("lodash.get");
-var iterableToString = require("./util/core/iterable-to-string");
-var objectProto = require("@sinonjs/commons").prototypes.object;
-var stringProto = require("@sinonjs/commons").prototypes.string;
-var typeOf = require("./util/core/typeOf");
-var valueToString = require("@sinonjs/commons").valueToString;
+var samsam = require("@sinonjs/samsam");
```

Comment:
Any reason to keep this file at all? Seems like it's obscuring what's really happening.